stimuli_ci_generation/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM stimuli_ci_base
```

Comment:
this will most likely not work here. @xuzhouyuan do you know where you built and registered `Dockerfile_rocker` and `Dockerfile_base`? I think if I know the image tag of `Dockerfile_base` in gcp registry, I can replace it here and it should work :-?

Or any recommendations?

Reply:
rocker and base are not registered with gcp, they need to be built locally before the app image can be built, locally. To use cloud build to build all three images automatically, I think we need to change the tags for stimuli_ci_rocker and stimuli_ci_base to gcr.io/[projectname]/[imagetag]

Reply:
I found a workaround :D I pushed our image to gcp container registry and is currently using it

Reply:
nice